### DISCUSSION TITLE
Inquiry about chemical potential value in BC in 2D CH model

### DISCUSSION THREAD

**Discussion by abc-hy** [idaholab/moose#20365] [Q&A Modules: Phase field] [answered]
Hi everyone,

I have some questions about how to set the chemical potential in CH model in Boundary condition. For concentration at boundary, it is simple, we only need to set the concentration we want at the boundary. However, for chemical potential, I was wondering what chemical potential expression in CH model is? What value should we set for the chemical potential at each boundary?
Anyone have any suggestions?

Thank you very much.
Best regards,

**Comment by laagesen** [comment, marked as answer]
Sorry for the slow reply on this. If you use the split Cahn-Hilliard formulation, where we solve for the chemical potential and concentration as two coupled variables, then chemical potential is one of your nonlinear variables and you can use any of the BC classes to set it as you like. We generally recommend the split CH formulation instead of the direct solution method because you get better numerical performance. More info about the differences can be found here:
https://mooseframework.inl.gov/modules/phase_field/Phase_Field_Equations.html
